content/en/docs/Getting Started/_index.md

```diff
@@ -217,7 +217,8 @@ rpm-based distros).
 
     ```bash
     # Replace LINK-TO-THE-PACKAGE with the package URL you would like to use.
-    rpm -Uvh LINK-TO-THE-PACKAGE
+    curl -LO LINK-TO-THE-PACKAGE
+    sudo apt install ./PACKAGE-NAME
```

Comment:
should this be `dpkg -i` instead of `apt install`?

Reply:
Yes, makes sense; using dpkg / rpm to install TektonCD CLI as a package and apt / dnf when installing from PPA / repository.